tests/test.sh

```diff
@@ -204,11 +224,21 @@ wait_for_completion() {
     for generator in ${generators[@]}; do
         status=$(get_pod_status ${generator})
         get_stats
+        if [[ $DIRECTION != "internal" ]]; then
```

Comment:
can you reverse logic ... it will make more sense

Reply:
done